Logger/LogFactory.cs

```diff
@@ -1,11 +1,27 @@
 ﻿namespace Logger
 {
+
     public class LogFactory
     {
+
+        private string _path;
```

Comment:
Fields should be underscore PascalCase
```suggestion
        private string _Path;
```